clients/venice-admin-tool/src/main/java/com/linkedin/venice/DumpAdminMessages.java

```diff
@@ -1,6 +1,6 @@
 package com.linkedin.venice;
 
-import static com.linkedin.venice.kafka.protocol.enums.MessageType.CONTROL_MESSAGE;
+import static com.linkedin.venice.kafka.protocol.enums.MessageType.*;
```

Comment:
Let's not use wildcard imports. You can use this to update your IDE configs: https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports

Reply:
SG.